message_sender/factory.py

```diff
@@ -462,4 +479,5 @@ def create_whatsapp_client(cls, channel):
             channel.configuration["API_TOKEN"],
             channel.configuration.get("HSM_NAMESPACE"),
             channel.configuration.get("HSM_ELEMENT_NAME"),
+            channel.configuration.get("MESSAGE_TTL"),
```

Comment:
We need to modify the `__init__` of the `WhatsAppApiSender` to accept this additional argument